src/measures.jl

```diff
@@ -83,6 +83,23 @@ function measure(t::Tetrahedron)
   abs((A - D) ⋅ ((B - D) × (C - D))) / 6
 end
 
+function measure(splx::Simplex{K,Dim,T,N}) where {K,Dim,T<:Real,N}
+  # https://en.wikipedia.org/wiki/Cayley%E2%80%93Menger_determinant#Definition.
+  dists = pairwise(SqEuclidean(), coordinates.(vertices(splx)))
+  mat = [  # somehow Julia fails the type inference for `mat` if we don't use the "nested" structure
+    [dists ones(T, size(dists, 1))];
+    [ones(T, size(dists, 2))' 0]
+  ]
+  if K <= 11  # Integer overflow for K>11
+    factor = (-1)^(K + 1) // (factorial(K)^2 * 2^K)
+    sqrt(factor * det(mat))
+  else
+    factor = (-1)^(K + 1) // (factorial(big(K))^2 * 2^big(K))
```

Comment:
Is this factor available as a floating point number in Combinatorics.jl or elsewhere? These special functions sometimes have special implementations to avoid these overflow issues.

Reply:
I haven't found this factor anywhere.

Reply:
I am starting to wonder if this high-dimensional Simplex is a good fit for Meshes.jl. The package was created to represent objects in the physical world up to 3 dimensions. I am feeling that your applications with high-dimensional distributions may deserve a dedicated package with symplectic algebra.

Initially we thought that it would be a good idea to add a new topological data structure for simplicial complexes. The new `Simplex` type was introduced to address a separate issue (higher-dimensional spaces). However, the type is overlapping with the existing `Segment`, `Triangle`, and `Tetrahedron`, and the implementations of `Simplex` methods will likely require additional dependencies, with specialized mathematical functions.

Let me think more about it before we continue working on this. Either way, you can easily move the code to a dedicated package for your specific needs.

Sorry for only noticing this additional requirements now.

Reply:
Alright, thanks for your thoughts. Let me know when you've had the time to think it through. When I was looking for a library to solve my problem, I arrived at Meshes.jl because [the overview page](https://juliageometry.github.io/MeshesDocs/dev/#Overview) does explicitly state that it can work with "general manifolds embedded in $\mathbb R^n$". In the comment above you write something different. Perhaps it is a good time to understand what this package tries and doesn't try to do.
I think merging this PR, aswell as the other two, would bring us closer to the goal described in the overview pages. However, if you do want to stay with "physical world" meshes, that's fair, and we can adjust the documentation, and perhaps split out the current functionality into a new package.

I am also available for a call if you think that would help. Thanks!

Reply:
Thank you @RomeoV for understanding it.  We need to make it more explicit in the docs that n = 1, 2, 3 in this project, and that a dedicated package would be better suited for n > 3.

I will go ahead and close the other PRs so that you can work on a separate repository to meet your needs with high-dimensional distributions. I will also remove the `Simplex` type from Meshes.jl and make it a breaking change.